core/circuitbreaker/circuit_breaker_test.go

```diff
@@ -29,7 +29,7 @@ func (m *CircuitBreakerMock) CurrentState() State {
 	return args.Get(0).(State)
 }
 
-// HandleCompleted handle the entry completed
+// OnRequestComplete handle the entry completed
 // rt: the response time this entry cost.
 func (m *CircuitBreakerMock) HandleCompleted(rt uint64, err error) {
```

Comment:
Also need to rename the HandleCompleted as OnRequestComplete in CircuitBreakerMock

Reply:
Updated, please check.